pages/agent/v3/elastic_ci_aws.md.erb

```diff
@@ -1,26 +1,21 @@
-# Elastic CI Stack for AWS Overview
+# Elastic CI Stack for AWS
 
-The Buildkite Elastic CI Stack for AWS gives you a private, autoscaling
-[Buildkite Agent](/docs/agent/v3) cluster. Use it to parallelize large test
-suites across hundreds of nodes, run tests and deployments for Linux or Windows
-based services and apps, or run AWS ops tasks.
-
-{:toc}
-
-
-## Quickstart
+The Buildkite Elastic CI Stack for AWS gives you a private, autoscaling [Buildkite agent](/docs/agent/v3) cluster. Use it to parallelize large test suites across hundreds of nodes, run tests and deployments for Linux or Windows based services and apps, or run AWS ops tasks.
 
 See the [Elastic CI Stack for AWS tutorial](/docs/tutorials/elastic_ci_stack_aws) for a step-by-step guide, or jump straight in:
 
-
 <!-- vale off -->
 <!-- alex ignore master -->
 
 <a href="https://console.aws.amazon.com/cloudformation/home#/stacks/new?stackName=buildkite&templateURL=https://s3.amazonaws.com/buildkite-aws-stack/latest/aws-stack.yml"><%= image "launch-stack.svg", alt: "Launch stack button" %></a>
 
 <!-- vale on -->
 
-> The stack creates its own VPC (virtual private cloud) by default. Best practice involves setting up a separate development AWS account and using role switching and consolidated billing. Refer to the [Delegate Access Across AWS Accounts tutorial](http://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html) for more information.
+{:toc}
+
+## Before you start
+
+> Elastic Stack for AWS creates its own VPC (virtual private cloud) by default. It's recommended to set up a separate development AWS account and use role switching and consolidated billing. You can check out this external tutorial for more information on how to ["Delegate Access Across AWS Accounts"](http://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html).
```

Comment:
I'd quite like to rephrase this "it's recommended" as well. Maybe just "Best practice is to set up..."